triggers/service/src/test/resources/logback.xml

```diff
@@ -11,6 +11,9 @@
     <logger name="io.grpc.netty" level="WARN">
         <appender-ref ref="stderr-appender"/>
     </logger>
+    <logger name="com.daml.lf.engine.trigger" level="DEBUG">
```

Comment:
Did you intend to keep this in? No issues with that, just checking.

Reply:
Good catch, thinking about it, it's been very useful to debug test failures, so seems sensible to keep enabled.